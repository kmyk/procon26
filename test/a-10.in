11111111111111111111111111111111
11111111111111111111111111111111
11111111111100111111111111111111
11000000111100111111111111111111
11111101111100111111111111111111
11111101111100111111111111111111
11111001111111111111111111111111
11111111111100000111111111111111
11111111111100000111111111111111
11101111111100000111111111111111
11000011111111111111111111111111
11110111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

3
01000000
11000000
00000000
00000000
00000000
00000000
00000000
00000000

00000000
11110000
00000000
00000000
00000000
00000000
00000000
00000000

11111100
01000000
01000000
01100000
00000000
00000000
00000000
00000000
