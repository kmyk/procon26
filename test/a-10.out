

0 3 T 0
